executor/executor.go

```diff
@@ -988,21 +1032,25 @@ func (e *HashSemiJoinExec) Next() (*Row, error) {
 		if e.anti && !isNull {
 			matched = !matched
 		}
-		if e.withAux {
+		// For auxMode subquery, we returns the row with a Datum indicates if it's match,
```

Comment:
->For the auxMode subquery, we return the row with a Datum indicating if it's a match.